wagtail/contrib/forms/views.py

```diff
@@ -68,6 +69,13 @@ class FormPagesListView(generic.IndexView):
     model = Page
     is_searchable = False
 
+    def get_breadcrumbs_items(self):
+        if not self.model:
+            return self.breadcrumbs_items
+        return self.breadcrumbs_items + [
+            {"url": "", "label": self.page_title, "sublabel": gettext("Pages")},
```

Comment:
Not using `page_subtitle` for the "Pages" part so that it's not added to the `<title>` tag, as per the behaviour before #11331.